src/meta/protos/proto/index.proto

```diff
@@ -51,8 +51,11 @@ message IndexMeta {
   string created_on = 3;
 
   // The time index dropped.
-  optional string drop_on = 4;
+  optional string dropped_on = 4;
+
+  // The time index updated.
+  optional string updated_on = 5;
 
   // The index based query string
-  string query = 5;
+  string query = 6;

```

Comment:
Never change the index of a field. Protobuf field is identified by the index.

Reply:
I made this change because the feature is not currently being used，if so, is it allowed?

Reply:
It would be OK if there is no data with `query = 5` that has ever been written to meta-service.